### PR TITLE
Fix description of Frequency, Shape, and Ramp Mode Parameters

### PR DESCRIPTION
**Pull request by Kautenja**
fix the description of Frequency and Ramp mode parameters. They were flip-flopped.

### CODE REVIEW

File: src/Tides2.cpp [hunk 1]
```diff
@@ -103,8 +103,8 @@ struct Tides2 : Module {
 		config(NUM_PARAMS, NUM_INPUTS, NUM_OUTPUTS, NUM_LIGHTS);
 		configParam(RANGE_PARAM, 0.0, 1.0, 0.0, "Frequency range");
 		configParam(MODE_PARAM, 0.0, 1.0, 0.0, "Output mode");
-		configParam(FREQUENCY_PARAM, -48, 48, 0.0, "Ramp mode");
-		configParam(SHAPE_PARAM, 0.0, 1.0, 0.5, "Frequency");
+		configParam(FREQUENCY_PARAM, -48, 48, 0.0, "Frequency");
+		configParam(SHAPE_PARAM, 0.0, 1.0, 0.5, "Ramp mode");
 		configParam(RAMP_PARAM, 0.0, 1.0, 0.0, "Shape");
```
Comment (SteveRussell33) [inline src/Tides2.cpp]: Doesn't Ramp and Shape need to be changed too?

Comment (Kautenja) [reply src/Tides2.cpp]: Oh, yes that's right.

Comment (Kautenja) [reply src/Tides2.cpp]: I added another commit to swap ramp mode and shape.

**Comment by AndrewBelt** [comment]
Thanks!

**Pull request merged**